NebulaPatcher/Patches/Dynamic/GameLoader_Patch.cs

```diff
@@ -0,0 +1,21 @@
+﻿using HarmonyLib;
+using NebulaModel.Logger;
+using System;
+using NebulaHost;
+using NebulaWorld;
+
+namespace NebulaPatcher.Patches.Dynamic
+{
+    [HarmonyPatch(typeof(GameLoader), "FixedUpdate")]
+    class GameLoader_Patch
+    {
+        public static void Postfix(int ___frame)
+        {
+            Log.Info("FixedUpdate: " + ___frame);
+            if(___frame >= 10)
+            {
+                SimulatedWorld.OnGameLoadCompleted();
```

Comment:
Will this be called multiple times after the frame 10 ?

Reply:
No I changed it to 11, since thats when the GameLoader self destructs